### ISSUE TITLE
202355599_강규석 질문있습니다.

### ISSUE THREAD

**Issue by Kangkyuseokk** [1004-head/WebBasic_23-1#4]
## 질문 요약
1. 4주차 과제 제출 방법

## 질문 상세
한문제씩 정답체크한 것을 모두 캡처본을 떠서 그 캡처본을 다 모아 pdf방식으로 제출하는 것 인가요..?
제출방식을 정확하게 잘 모르겠어서 질문 드립니다.

**Comment by 1004-head** [comment]
네 맞습니다 한문제씩 체크한 화면 캡처하고 마지막 결과까지 캡처해서 하나의 pdf 파일로 모은 후 그 pdf파일을 제출하시면 됩니다.
